SC-workflow/Persist-Seq_v0.3.5/Galaxy-Workflow-Persist-Seq_v0.3.5,_Scanpy_1.8.1_(cell_cycle).ga

```diff
@@ -1280,12 +1258,7 @@
                     "output_name": "output_h5ad"
                 }
             },
-            "inputs": [
-                {
-                    "description": "runtime parameter for tool Scanpy ScaleData",
-                    "name": "input_obj_file"
-                }
-            ],
+            "inputs": [],
```

Comment:
This change looks weird, could you look at this side to side in both workflows, seems like a missing input here perhaps for ScaleData?